client/src/Components/PreNavBar.js

```diff
@@ -3,15 +3,23 @@ import { Link } from "react-router-dom";
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Button from '@material-ui/core/Button';
+import Logout from './LogNavBar'
 
 import Logo from "./Logo";
 
-function PreNavBar() {
-  return (
-    <div>
-      <AppBar position="static">
+function PreNavBar(props) {
+  const handleLogout = e => {
```

Comment:
Since this is the Pre Auth (pre sign in) navigation bar, there should not be a way to log out here.